inference-engine/src/inference_engine/src/threading/ie_parallel_custom_arena.cpp

```diff
@@ -73,37 +74,87 @@ static bool is_binding_environment_valid() {
 #        endif /* _WIN32 && !_WIN64 */
 }
 
-static int numa_nodes_count = 0;
-static int* numa_nodes_indexes = nullptr;
-
-static int core_types_count = 0;
-static int* core_types_indexes = nullptr;
-
-static void initialize_system_topology() {
-    static std::once_flag is_topology_initialized;
-
-    std::call_once(is_topology_initialized, [&] {
-        if (is_binding_environment_valid()) {
+class TBBSystemTopology {
```

Comment:
Want to propose to specify that this object is unrelated to TBB in common sense. Please consider changing the prefix from "TBB" to "TBBbind"

Reply:
ok